website/docs/cloud-docs/vcs/bitbucket-server.mdx

```diff
@@ -91,7 +91,9 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
    ![Bitbucket Server screenshot: The application links page](/img/docs/bitbucket-server-application-links.png)
 
-1. Enter Terraform Cloud's URL in the text field (`https://app.terraform.io`, or the hostname of your Terraform Enterprise instance) and click the "Create new link" button.
+1. In application links, select Create link.
+2. Select Atlassian product as the link type. This option also works for external applications. This is the original application links mechanism – Atlassian has left it under this option so users can still use their OAuth 1.0 integrations.
```

Comment:
I'm wondering if we can shorten that even more. I want to phrase this in terms of what this lets users do and address the user as "you" rather than talking about "users" more broadly. Thoughts? 

@mgarrell777 - Feel free to weigh in here too! I took you original edit above and put it here. :) 
```suggestion
1. Select **Create link** in application links.
2. Select **Atlassian product** as the link type. This option also works for external applications lets you continue to use OAuth 1.0 integrations.
```